### PR TITLE
Exposing schema parent information

### PR DESCRIPTION
**Pull request by zimnx**

### CODE REVIEW

File: extension/goplugin/schemas.go [hunk 1]
```diff
@@ -523,6 +523,11 @@ func (schema *Schema) Properties() []goext.Property {
 	return properties
 }
 
+// Parents returns schema_ids which given schema extends
+func (schema *Schema) Parents() []string {
+	return schema.raw.Extends
```
Comment (przemyslaw-dobrowolski-cl) [inline extension/goplugin/schemas.go]: In gohan, it it named 'extends' but in the interface it is parent. I think it's an inconsistency that should be fixed.

File: extension/goext/schemas.go [hunk 2]
```diff
@@ -187,6 +187,9 @@ type ISchema interface {
 
 	// Properties returns properties of schema
 	Properties() []Property
+
+	// Parents return list of schema_ids which given schema extends
+	Parents() []string
```
Comment (przemyslaw-dobrowolski-cl) [inline extension/goext/schemas.go]: What is the rationale tor this new addition?

Comment (zimnx) [reply extension/goext/schemas.go]: Register event handlers only if schema extends some other schema. There is no way to check it right now

File: extension/goplugin/schemas.go (hunk 1 above)
Comment (zimnx) [reply extension/goplugin/schemas.go]: I've changed that to keep consistency, but Extends doesn't fit to me at all.

File: extension/goext/schemas.go (hunk 2 above)
Comment (jkmar) [reply extension/goext/schemas.go]: What about this: #541?

Comment (zimnx) [reply extension/goext/schemas.go]: #541 gives relation info but in other direction.

**Pull request merged**